### PR TITLE
Implement account details view

### LINKED ISSUE #5238: Move account specific information to account menu
### Acceptance Criteria

Add the following options to account
- Copy account
- View account

### PR DESCRIPTION
**Pull request by ikem-legend**
### What was the problem?

This PR resolves #5238

### How was it solved?

- [x] Update account menu
- [x] Implement account details UI

### How was it tested?

- Visually
- Unit tests

### CODE REVIEW

File: src/modules/wallet/components/AccountDetails/index.js [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+  }, [authData?.data]);
+
+  const {
+    register,
+    watch,
+    handleSubmit,
+    reset,
+    formState: { errors, isDirty },
+    setValue,
+    setError,
```
Comment (eniolam1000752) [inline src/modules/wallet/components/AccountDetails/index.js]:
```suggestion
    setValue,
    setError,
    formState: { errors, isDirty },
```

**Review by eniolam1000752 — CHANGES_REQUESTED** [review]
## Ui issues observed

- The tooltip is clipped due tot he overflow on the section
image


- There is suppose to be a padding of 8px in the highlighted space
image

File: src/modules/wallet/components/AccountDetails/index.js [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+  })
+  .required();
+
+const AccountDetails = () => {
+  const history = useHistory();
+  const dispatch = useDispatch();
+  const { t } = useTranslation();
+  const [currentAccount] = useCurrentAccount();
+  const { accounts } = useAccounts();
+  const queryAddress = selectSearchParamValue(history.location.search, 'validatorAddress');
```
Comment (ManuGowda) [inline src/modules/wallet/components/AccountDetails/index.js]:
```suggestion
  const queryAddress = selectSearchParamValue(history.location.search, 'validatorAddress') || selectSearchParamValue(history.location.search, 'address');
```

File: src/modules/wallet/components/AccountDetails/index.js [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+                  className={styles.rowValue}
+                  text={truncateAddress(address)}
+                />
+              </div>
+              <div className={styles.row}>
+                <span className={styles.title}>{t('Public Key')}:</span>
+                <CopyToClipboard
+                  value={publicKey}
+                  className={styles.rowValue}
+                  text={truncateAddress(publicKey)}
```
Comment (ManuGowda) [inline src/modules/wallet/components/AccountDetails/index.js]:
```suggestion
                  value={publicKey || currentAccount.metadata.pubkey}
                  className={styles.rowValue}
                  text={truncateAddress(publicKey || currentAccount.metadata.pubkey)}
```

File: src/modules/wallet/components/AccountDetails/index.js (hunk 1 above)
Comment (ikem-legend) [reply src/modules/wallet/components/AccountDetails/index.js]: Fixed

File: src/modules/wallet/components/AccountDetails/index.js (hunk 3 above)
Comment (ikem-legend) [reply src/modules/wallet/components/AccountDetails/index.js]: Fixed

**Review by ManuGowda — DISMISSED** [review]

**Review by eniolam1000752 — CHANGES_REQUESTED** [review]
There is just one minor issue, The page goes blank when there are more than 4 signatories to the account
image

**Review by eniolam1000752 — APPROVED** [review]

**Review by Balanced02 — APPROVED** [review]

**Pull request merged**